src/Symfony/Component/Yaml/Yaml.php

```diff
@@ -30,6 +30,7 @@ class Yaml
     const DUMP_MULTI_LINE_LITERAL_BLOCK = 128;
     const PARSE_CONSTANT = 256;
     const DUMP_EMPTY_ARRAY_AS_SEQUENCE = 1024;
+    const PARSE_KEYS_AS_STRING = 2048;
```

Comment:
But sometimes you are not able to control how the actual YAML structure looks like. I don't think we should make it harder to parse such files.